network-api/networkapi/mozfest/templates/partials/signup.html

```diff
@@ -0,0 +1,22 @@
+<!--
```

Comment:
If this is something that might be used in the future let's we add a comment here. If not, let's remove this chunk of code.